src/Infrastructure.EntityFramework/EntityFrameworkCache.cs

```diff
@@ -0,0 +1,304 @@
+﻿using Bit.Infrastructure.EntityFramework.Models;
```

Comment:
Explanation: IDistributedCache implementation for EF. Mostly copies the SqlServerCache implementation from Microsoft here: https://github.com/dotnet/aspnetcore/blob/main/src/Caching/SqlServer/src/SqlServerCache.cs